Formula/node.rb

```diff
@@ -56,30 +56,11 @@ def install
 
     if build.with? "npm"
       resource("npm").stage buildpath/"npm_install"
-
-      # make sure npm can find node
-      ENV.prepend_path "PATH", bin
```

Comment:
>Reading your other review comment I guess this issue is more likely caused by the new npm installtion logic in this formula and not by the necessarry changes to Language::node to make std_npm_install_argscompatible with npm@5.

Yep, I agree that is more likely than not.

> Maybe the discussion about the necessary changes to Language::node should be separated over to an issue/pr in Homebrew/brew.

I'm sure Dom will do just that :)